sshbuf-getput-basic.c

```diff
@@ -274,6 +274,11 @@ sshbuf_get_cstring(struct sshbuf *buf, char **valp, size_t *lenp)
 	const u_char *p, *z;
 	int r;
 
+	if(p == NULL)
+	{
+		p = 0;
+	};
```

Comment:
The C standard says "0"  in pointer context is a null pointer so I have no idea what you're trying to achieve with this.